openmrs/apps/reports/VIRAL_SUPPRESSION/sql/indicator1a__viral_suppression_routine.sql

```diff
@@ -0,0 +1,27 @@
+SELECT 'Routine' AS 'Number of ART patients with suppressed VL results (<1,000 copies/ml) documented in the medical or laboratory records/LIS within the past 12 months' ,
+            viralSuppression_Indicator1c('#startDate#','#endDate#', 0, 1, 0, 'M') AS '<1 M',
+            viralSuppression_Indicator1c('#startDate#','#endDate#', 0, 1, 0, 'F') AS '<1 F',
```

Comment:
Hi @tmvumbi2 ,
Here you call a wrong function, It should be "viralSuppression_Indicator1a".